src/components/navigation/constants.ts

```diff
@@ -1,6 +1,6 @@
 export const navLinks = [
   { href: "/about", name: "About" },
   { href: "/shop", name: "Shop" },
-  { href: "/contact", name: "Contact" },
+  { href: "/#contacts", name: "Contacts" },
```

Comment:
Thats great, but you can improve it like this:
```suggestion
  { href: "#contacts", name: "Contacts" },
```